perf/istio-install/istioctl_profiles/istio-non-sds.yaml

```diff
@@ -1,10 +1,9 @@
-apiVersion: install.istio.io/v1alpha2
-kind: IstioControlPlane
+apiVersion: operator.istio.io/v1alpha1
```

Comment:
Will these changes continue to work with release 1.4?

Reply:
this is for master. 1.4 is still using the old API